source/base/ReferenceCounter.ooc

```diff
@@ -22,6 +22,7 @@ ReferenceCounter: class {
 			}
 		}
 	}
+	count ::= this _count
```

Comment:
`isSafe` is a property, not a member. A specific `isSafe` value isn't stored anywhere.